src/validators/recordCompleteness.ts

```diff
@@ -28,16 +28,16 @@ export function validateAndComputeRecordCompleteness(input: (TechRecordPut | Tec
 
 const validateSkeletonSchema = (input: (TechRecordPut | TechRecordGet), method: HttpMethod): boolean => {
   const isSkeletonSchema = identifySchema(input.techRecord_vehicleType as VehicleType, RecordCompleteness.SKELETON, method);
-  return isSkeletonSchema ? isValidObject(isSkeletonSchema, input) : false;
+  return isSkeletonSchema ? isValidObject(isSkeletonSchema, input) as boolean : false;
```

Comment:
why do you have to cast it as a boolean?
if what you want to check is a make a truthy/falsy value a boolean, use:
```suggestion
  return isSkeletonSchema ? !!isValidObject(isSkeletonSchema, input) : false;
```

Reply:
isValidObject can return an error object if asked for [https://github.com/dvsa/cvs-type-definitions/blob/develop/schema-validator.ts#L12](https://github.com/dvsa/cvs-type-definitions/blob/develop/schema-validator.ts#L12 ), even though here it will always return boolean, typescript was crying about it

Reply:
fair enough, might try some typescript magic to make it so we don't have to do that